.github/workflows/tests.yaml

```
@@ -9,8 +9,8 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        os: [ ubuntu-latest, macos-latest ]
-        php: [ 8.1, ]
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
```

```suggestion
        os: [ ubuntu-latest ]
```
let's drop MacOS and Windows